omnibus/files/private-chef-cookbooks/private-chef/libraries/private_chef.rb

```diff
@@ -780,9 +780,13 @@ def migrate_keys
       credentials.save if did_something
     end
 
+    def set_ldap_enabled
+      PrivateChef["ldap"]["enabled"] = !( PrivateChef["ldap"].nil? || PrivateChef["ldap"].empty? )
```

Comment:
Good call, also since we set it ldap to Mash.new above, I'm not really sure we need the nil check.